src/components/data/SongTable/UpcomingSongList.tsx

```diff
@@ -176,17 +175,21 @@ export const UpcomingSongList = ({
   const list = React.useRef<any>(null);
 
   useEffect(() => {
-    // Hint at variablesizelist to recheck row heights after queue length changed
+    // Hint at VariableSizeList to recheck row heights after queue length changed
     list.current?.resetAfterIndex(queue.length);
   }, [queue.length]);
 
   const frameRef = useContext(FrameRef);
-  const onScroll = useCallback(({ scrollTop }) => {
-    list.current?.scrollTo(scrollTop);
-  }, []);
+  const onScroll = useCallback(
+    ({ scrollTop }: { scrollLeft: number; scrollTop: number }) => {
+      list.current?.scrollTo(scrollTop);
+    },
+    []
+  );
 
   const renderList = useCallback(
     ({ height, width }: { height: number; width: number }) => (
+      // @ts-ignore React 18 typings issue
```

Comment:
react-virtualized wasn't updated for quite some time, no support for React 18 typings

Reply:
🤔 https://github.com/DaPulse/react-virtualized